classes/class-wpcom-liveblog-entry.php

```diff
@@ -281,6 +291,32 @@ private static function user_object_from_comment_id( $comment_id ) {
 		}
 		return $user_object;
 	}
+
+	/**
+	 * Handles stripping out any Restricted Shortcodes and replacing them with the
+	 * preconfigured string entry.
+	 *
+	 * @param array $args The new Live blog Entry.
+	 * @return mixed
+	 */
+	public static function handle_restricted_shortcodes( $args ) {
+
+		// Runs the restricted shortcode array through the filter to modify it where applicable before being applied.
+		self::$restricted_shortcodes = apply_filters( 'liveblog_entry_restrict_shortcodes', self::$restricted_shortcodes );
+
+		// Foreach lookup key, does it exist in the content.
+		foreach ( self::$restricted_shortcodes as $key => $value ) {
+
+			// Regex Pattern will match all shortcode formats.
+			$pattern = "/\[{$key}(.*?)?\](?:(.+?)?\[\/{$key}(.*?)?\])?/";
```

Comment:
We should replace this with `get_shortcode_regex()` so that we're using whatever core uses to find shortcodes.